src/screens/ActivitiesPage.js

```diff
@@ -11,52 +22,64 @@ const ActivitiesPage = () => {
       spacing={2}
       p={8}
     >
+      
+      {signin ?
+        <Grid item xs={12}>
+          <Typography variant="subtitle2" align='center'>Please complete 5 activities for the month</Typography>
+          <LinearProgressWithLabel variant='determinate' value={progress} />
```

Comment:
Is it possible to have the label say 4/5 instead of a percentage? Might have to use LinearProgress no label and add the text label yourself to the right. This means instead of progress, we'd use a state of activityCount which would be a number out of 5. then in value we would pass in activityCount * 20 to get the percentage.